packages/ingest-mongodb-public/src/sources/snooty/SnootyDataSource.ts

```diff
@@ -1,9 +1,9 @@
 import { createInterface } from "readline";
 import { Page, PageFormat, logger } from "mongodb-rag-core";
 import fetch from "node-fetch";
-import { DataSource } from "../DataSource";
+import { DataSource } from "mongodb-rag-ingest/sources";
```

Comment:
```suggestion
import { DataSource, ProjectBase } from "mongodb-rag-ingest/sources";
```

Can combine this import with the one two lines below